framework/src/modules/http_api/controllers/accounts.js

```diff
@@ -42,7 +42,6 @@ function accountFormatter(totalSupply, account) {
 
 	if (account.isDelegate) {
 		formattedAccount.delegate = _.pick(account, [
-			'username',
```

Comment:
not related to this PR, but can you also remove the `voteWeight` below?
Also, `formattedAccount.delegate.voteWeight,` at line 55 below to `formattedAccount.totalVotesReceived,`?